templates/taxi/car_detail.html

```diff
@@ -1,12 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>Car Details</h1>
+  <p>Id {{ car.id }}</p>
```

Comment:
The car ID is displayed correctly. Ensure that all necessary car details are displayed as required by the task description.